### PR TITLE
Added additionalSkills, adhoc items, code review, junit, postman test

### PR DESCRIPTION
**Pull request by anitha-srinivasan-egov**
1. Added skills, dummy user in additionalDetails  as json object in response and fetch it from request in create and update
2. updated workflow - removed 'closedAfterRejection' and 'Closed'
3. added state level tenant search
4. Removed assignees from request
5. Junit test classes for create and estimate api
6. postman collection with test scripts
7.  Code review comments - column name, workflow businessService , method names, loggers, comments, removed workflow action check in update, added 'commuteAttendance' flag check

### CODE REVIEW

File: backend/muster-roll-service/src/main/java/org/egov/validator/MusterRollValidator.java [hunk 1]
```diff
@@ -46,6 +46,8 @@ public class MusterRollValidator {
      * @param musterRollRequest
      */
     public void validateEstimateMusterRoll(MusterRollRequest musterRollRequest){
+        log.info("MusterRollValidator::validateEstimateMusterRoll");
```
Comment (subhashini-egov) [inline backend/muster-roll-service/src/main/java/org/egov/validator/MusterRollValidator.java:49]: @Anitha-Srinivasan - please add the muster roll number for which this request is coming in. Else, we can't debug it later.

File: backend/muster-roll-service/src/test/resources/MusterRollMDMSData.json [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+          "code": "ROUND_OFF_HOURS",
+          "value": "true"
+        },
+        {
+          "code": "Skill 1",
+          "value": "Unskilled"
+        },
+        {
+          "code": "Skill 2",
+          "value": "Semiskilled"
```
Comment (subhashini-egov) [inline backend/muster-roll-service/src/test/resources/MusterRollMDMSData.json:36]: Semi-skilled

File: backend/muster-roll-service/src/test/resources/MusterRollMDMSData.json [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+        },
+        {
+          "code": "Skill 3",
+          "value": "Skilled"
+        },
+        {
+          "code": "Skill 4",
+          "value": "Highskilled"
```
Comment (subhashini-egov) [inline backend/muster-roll-service/src/test/resources/MusterRollMDMSData.json:44]: High-Skilled

**Review by subhashini-egov — APPROVED** [review]
@anitha-srinivasan-egov , added a few comments. But I will go ahead and merge this. Please make sure to do those changes.

File: backend/muster-roll-service/src/main/java/org/egov/validator/MusterRollValidator.java (hunk 1 above)
Comment (anitha-srinivasan-egov) [reply backend/muster-roll-service/src/main/java/org/egov/validator/MusterRollValidator.java:49]: I have logged the musterRollNumber of the incoming request for update. Will raise a separate PR along with other changes.

File: backend/muster-roll-service/src/test/resources/MusterRollMDMSData.json (hunk 2 above)
Comment (anitha-srinivasan-egov) [reply backend/muster-roll-service/src/test/resources/MusterRollMDMSData.json:36]: Updated in mdms file and deployed

File: backend/muster-roll-service/src/test/resources/MusterRollMDMSData.json (hunk 3 above)
Comment (anitha-srinivasan-egov) [reply backend/muster-roll-service/src/test/resources/MusterRollMDMSData.json:44]: Updated in mdms file and deployed

**Pull request merged**